doc/reference/configuration/configuration_reference.rst

```diff
@@ -2877,6 +2878,120 @@ This section describes configuration parameters related to roles.
 
     **Since:** :doc:`3.0.0 </release/3.0.0>`.
 
+    Specify the roles of an instance.
+    To specify a role's configuration, use the :ref:`roles_cfg <configuration_reference_roles_cfg>` option.
+
+    There are two types of application roles:
+
+    -   Built-in roles. For example, the `CRUD <https://github.com/tarantool/crud>`__ module provides the ``roles.crud-storage`` and ``roles.crud-router`` roles that enable CRUD operations in a sharded cluster.
+    -   Custom roles. For example, you create a custom role to define a stored procedure or implement a supplementary service, such as an e-mail notifier or a replicator.
```

Comment:
AFAIK, we use the term 'built-in' for things that are built into tarantool executable. `crud` is an external module in this sense.

I would draw the line in a bit different way.

* Roles that are shipped by a Lua rock or provided by tarantool itself. The idea here is to write them once and use in different applications.
* Roles that are part of an application. The idea is to highlight that the given code  is a kind of service and, also, give a way to enable/disable this specific code for an administrator.